### PR TITLE
Previous care

### PR DESCRIPTION
**Pull request by gmiak**
Simple GUI for Previous Care Screen
- No logic is involved

### CODE REVIEW

File: androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/journal/triage/previousCare/PreviousCareContent.kt [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.EENX15_22_17.digital_journal.android.ui.components.*
+import se.predicare.journal.screens.PreviousCareDto
+
+@Composable
+fun previousHealthCare(item: PreviousCareDto) {
```
Comment (rickardlantz) [inline androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/journal/triage/previousCare/PreviousCareContent.kt:13]: Rendered composables should start with uppercase letter, the IDE should have shown a hint about this

File: androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/journal/triage/previousCare/PreviousCareScreen.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    onMenuClicked: () -> Unit
+) {
+    DetailPageWrapper(
+        title = stringResource(id = R.string.previousCare),
+        titleColor = Colors.triagePrimary,
+        onBackClicked = onBackClicked,
+        onMenuClicked = onMenuClicked
+    ) {
+        Column(modifier = Modifier.padding(top=20.dp)) {
+            TitledSectionTemp(title = stringResource(id = R.string.previousHealtCare)) {
```
Comment (rickardlantz) [inline androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/journal/triage/previousCare/PreviousCareScreen.kt:28]: Why do we have two types of titled section? we should merge them, or if they have different purposes, rename them.

File: androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/components/Radiobuttons.kt [hunk 3]
```diff
@@ -151,3 +155,45 @@ fun <E : Enum<*>> EnumRadioButtonWithTextField(
...
+    }
+
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Start
+    ) {
+        Box(contentAlignment = Alignment.Center) {
+            Box(Modifier
+                .size(checkboxBoxBackgroundSize)
+                .background(colorCheckBoxBeta, checkboxBackgroundShape)
```
Comment (rickardlantz) [inline androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/components/Radiobuttons.kt:181]: I think radiobuttons should still have the circle backgrounds

**Review by rickardlantz — APPROVED** [review]
GUI looks great! However, some code style issues and copy-paste leftovers

**Review by axelhertzberg — APPROVED** [review]
Everything is working good and looks nice, Good job.
We should use One *TitledSection* i the app so change that if you can.

**Pull request merged**